crates/re_viewer/src/misc/caches/tensor_decode_cache.rs

```diff
@@ -0,0 +1,127 @@
+use re_log_types::component_types::{Tensor, TensorId, TensorTrait};
+
+#[derive(thiserror::Error, Clone, Debug)]
+pub enum TensorDecodeError {
+    // TODO(jleibs): It would be nice to just transparently wrap
+    // `image::ImageError` and `tensor::TensorImageError` but neither implements
+    // `Clone`, which we need if we ant to cache the Result.
```

Comment:
```suggestion
    // `Clone`, which we need if we want to cache the Result.
```

isn't the later our own and we could implement Clone?